### PR TITLE
Docs: Relabel Localization as Explanation (Diátaxis)

### PR DESCRIPTION
**Pull request by benjaoming**
Refs: #9746 

---
:books: Documentation previews :books:

- User's documentation (`docs`): https://docs--9790.org.readthedocs.build/en/9790/

- Developer's documentation (`dev`): https://dev--9790.org.readthedocs.build/en/9790/

### CODE REVIEW

**Comment by benjaoming** [comment]
@ericholscher it's possible to have a lot of improvements here. But I feel like this is a large subject that we might want to push to later iterations?

One of the improvements would be a diagram for the translation workflow. We already have "add a diagram" opened as a suggestion in https://github.com/readthedocs/readthedocs.org/pull/9675 and I think we should be careful about how many times we want diagrams added before we should decide on an approach to easily have Mermaid diagrams added (it eventually wasn't that easy in my first attempt because of small technical reasons).

**Comment by ericholscher** [comment]
Yea, I think a small update here is best. We don't get a ton of users using this feature, so I'm less worried about it's overall quality.

File: docs/user/localization.rst [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+In this article, we explain high-level approaches to internationalizing and localizing your documentation.
 
-Read the Docs supports hosting your docs in multiple languages.
-There are two different things that we support:
+By default, Read the Docs assumes that your documentation is or *might become* multilingual one day.
+The initial default language is English and
+therefore you often see the initial build of your documentation published at ``/en/latest/``,
+where the ``/en`` denotes that it's in English.
+By having the ``en`` URL component present from the beginning,
+you are ready for the eventuality that you would want a second language.
```
Comment (ericholscher) [inline docs/user/localization.rst:11]: This is a great intro 👍

File: docs/user/localization.rst [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
 
-* A single project written in another language
-* A project with translations into multiple languages
+Read the Docs supports hosting your documentation in multiple languages.
```
Comment (ericholscher) [inline docs/user/localization.rst:13]:
```suggestion
Read the Docs supports hosting your documentation in multiple languages.
Read below for the various approaches that we support.
```

A bit short and not fully explained?

File: docs/user/localization.rst [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
 
+.. contents::
+    :local:
```
[the remark this answers is not in the document]
Comment (ericholscher) [reply docs/user/localization.rst:16]: Seems fine.

File: docs/user/localization.rst [hunk 4]
```diff
@@ -22,10 +30,13 @@ The language will be represented in the URL for your project.
 For example,
 a project that is in Spanish will have a default URL of ``/es/latest/`` instead of ``/en/latest/``.
 
-.. note:: You must commit the ``.po`` files for Read the Docs to translate your documentation.
+Projects with multiple translations (Sphinx)
+--------------------------------------------
```
Comment (ericholscher) [inline docs/user/localization.rst:34]:
```suggestion
Projects with multiple translations (Sphinx-only)
-------------------------------------------------
```

File: docs/user/localization.rst [hunk 5]
```diff
@@ -44,7 +55,7 @@ This has the results of serving:
 * ``phpmyadmin`` at ``http://phpmyadmin.readthedocs.io/en/latest/``
 * ``phpmyadmin-spanish`` at ``http://phpmyadmin.readthedocs.io/es/latest/``
 
-It also gets included in the Read the Docs flyout:
+It also gets included in the Read the Docs :term:`flyout menu`:
```
Comment (ericholscher) [inline docs/user/localization.rst:58]: :100:

**Review by ericholscher — APPROVED** [review]
Looks great 👍

File: docs/user/localization.rst (hunk 2 above)
Comment (benjaoming) [reply docs/user/localization.rst:13]: Yes, it's nice with the extra sentence here :100:

File: docs/user/localization.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (benjaoming) [reply docs/user/localization.rst:17]: Great, thanks for looking, I think it works well like this, too, just really needed a second opinion :+1: :heart:

File: docs/user/localization.rst (hunk 1 above)
Comment (benjaoming) [reply docs/user/localization.rst:11]: Thanks :blush: 

I hope we can have some time to expand fresh ideas in this section, but I think we should do it in another iteration. Will mark for later :+1:

**Pull request merged**